kube-aws-updater

```diff
@@ -102,38 +102,58 @@ if [[ -n "${aws_profile}" ]]; then
   fi
 fi
 
-echo "kube cluster :: ${kube_context}"
+function fail() {
+  echo $1 >&2
+  exit 1
+}
+
+function retry() {
+  local n=1
+  local max=5
+  local delay=4
+  while true; do
+    "$@" && break || {
+      if [[ $n -lt $max ]]; then
+        ((n++))
+        echo "command failed: attempt=$n max=$max"
+        sleep $delay;
+      else
+        fail "the command has failed after $n attempts."
```

Comment:
missing a `break` here? Else you will be doing this until the command succeeds, so there is no point for this `max` tries handling.